examples/own_dataset/README.md

```diff
@@ -1,4 +1,10 @@
 # Example of using your own dataset
+
+This example shows how to train your own dataset stored in CSV format.
```

Comment:
... how to train models *with* your own dataset ...

Reply:
done